CodeForPhilly/third-places
# Add env sample

# Overview

Adds a `.env.sample` file that can be duplicated and renamed to `.env` to add that functionality for both `docker-compose` and setting environment variable in general

Closes #19 

# Testing instructions

- In this branch, duplicate the `.env.sample` file and rename it to `.env`
- Ensure the setup instructions match that in the README
- Run `docker-compose up --build` and ensure:
  - [ ] The app builds successfully
  - [ ] The app runs successfully
  - [ ] You can naavigate to `localhost:4321` and 'localhost:8321/api/`
  - [ ] No errors occur

# Checklist before merge
